src/components/App.js

```diff
@@ -6,7 +6,7 @@ import ButtonPannel from './ButtonPannel';
 function App() {
   return (
     <>
-      <Display result={0} />
+      <Display />
       <ButtonPannel />
     </>
```

Comment:
- [ ] I noticed that you did not add the calculate.js import. Kindly import calculate.js in your `App` component as required.